### PR TITLE
Update Raspbian file names and NOOBS version

### PR DESCRIPTION
**Pull request by XECDesign**

### CODE REVIEW

**Comment by JamesH65** [comment]
This seems odd to be using zip files - can `dd` handle those?

**Comment by XECDesign** [comment]
No, I'm just an idiot. One second.

**Comment by XECDesign** [comment]
Should be okay now. Well spotted, thanks.

File: installation/installing-images/linux.md [hunk 1]
```diff
@@ -31,9 +31,9 @@
 
 In Linux it is possible to combine the unzip and SD copying process into one command, which avoids any issues that might occur when the unzipped image is larger than 4GB. This can happen on certain filesystems that do not support files larger than 4GB (e.g. FAT), although it should be noted that most Linux installations do not use FAT and therefore do not have this limitation.
 
-The following command unzips the zip file (replace 2020-08-20-raspios-buster.zip with the appropriate zip filename), and pipes the output directly to the dd command. This in turn copies it to the SD card, as described in the previous section.
+The following command unzips the zip file (replace 2020-08-20-raspios-buster.img with the appropriate zip filename), and pipes the output directly to the dd command. This in turn copies it to the SD card, as described in the previous section.
 ```
-unzip -p 2020-08-20-raspios-buster.zip | sudo dd of=/dev/sdX bs=4M conv=fsync
+unzip -p 2020-08-20-raspios-buster.img | sudo dd of=/dev/sdX bs=4M conv=fsync
 ```
```
Comment (JamesH65) [inline installation/installing-images/linux.md:37]: armhf?

Comment (JamesH65) [reply installation/installing-images/linux.md:37]: And in paragraph above.

**Comment by lurch** [comment]
LOL, I've just realised in hindsight that this PR is misnamed :laughing: 
Should have been "Update Raspberry Pi OS file names and NOOBS version".

**Comment by XECDesign** [comment]
> LOL, I've just realised in hindsight that this PR is misnamed

I have to admit none of this has been my best work.

**Comment by JamesH65** [comment]
:-o

**Pull request merged**